docs/source/reference-lowlevel.rst

```diff
@@ -56,6 +56,35 @@ Global statistics
 .. autoclass:: RunStatistics()
 
 
+.. _trio_contexts:
+
+The current Trio context
+------------------------
+
+There are two different types of contexts in :mod:`trio`. Here are the
+semantics presented as a handy table. Choose the right function for
+your needs.
+
++---------------------------------+-----------------------------------+------------------------------------+
+| situation                       | :func:`trio.lowlevel.in_trio_run` | :func:`trio.lowlevel.in_trio_task` |
++=================================+===================================+====================================+
+| inside a running async function | `True`                            | `True`                             |
```

Comment:
might want to say "Trio-flavored async function", since with guest mode you could have other-loop async functions

Reply:
I'm not sure where we define "Trio-flavored" but at the very least it's clear from context so I'll do it.